test/JuMPTest/JuMPTest.jl

```diff
@@ -23,6 +23,106 @@ function jump_luksan_vlcek_model(N)
 
     return jm
 end
+function fixed_variable_e2etest()
+    N=5
+    jm = JuMP.Model()
+
+    JuMP.@variable(jm, x[1:N])
+    JuMP.fix(x[1], 1.0)
+    JuMP.@constraint(jm, sum(x) == 1.0)
+    JuMP.@objective(jm, Min, sum(2*x[i]^2 for i in 1:N))
+
+    em = ExaModel(jm)
+    @test only(em.meta.lcon) == only(em.meta.ucon) == 1.0
+
+    # FIXME: two ConstraintAug?
+    wcons = em.cons
+    @test typeof(wcons) <: ExaModels.ConstraintAug
+    @test typeof(wcons.f.f) <: ExaModels.Null
+
+    cons = wcons.inner
+    @test typeof(cons) <: ExaModels.ConstraintAug
+    @test typeof(cons.f.f) <: Pair
+
+    @test typeof(cons.f.f.second) <: ExaModels.Node2{typeof(*), ExaModels.Var{T1}, T2} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
+
+    @test typeof(cons.inner) <: ExaModels.Constraint
+    @test typeof(cons.inner.f.f) <: ExaModels.Null{Nothing}
+    @test typeof(cons.inner.inner) <: ExaModels.ConstraintNull
+
+    @test typeof(em.objs.f.f) <: ExaModels.Null
+    @test typeof(em.objs.inner.f.f) <: ExaModels.Node2{typeof(*), T1, ExaModels.Node1{typeof(abs2), ExaModels.Var{T2}}} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
+
+    jm = JuMP.Model()
+
+    JuMP.@variable(jm, x[1:N])
+    JuMP.@variable(jm, p in JuMP.Parameter(1.0))
+    JuMP.@constraint(jm, sum(x) == p)
+    JuMP.@objective(jm, Min, sum(x))
+
+    @test_broken em = ExaModel(jm) # FIXME: merge #158
+    # @test only(em.meta.lcon) == only(em.meta.ucon) == 0.0
+    # @test only(em.θ) == 1.0
+    # wcons = em.cons
+    # @test typeof(wcons) <: ExaModels.ConstraintAug
+    # @test typeof(wcons.f.f) <: ExaModels.Null
+    # cons = wcons.inner
+    # @test typeof(cons) <: ExaModels.ConstraintAug
+    # @test typeof(cons.f.f) <: Pair
+    # @test typeof(cons.f.f.second) <: ExaModels.Node2{typeof(*), ExaModels.ParameterNode{T1}, T2} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
+    # @test typeof(cons.inner) <: ExaModels.ConstraintAug
+    # @test typeof(cons.inner.f.f) <: Pair
+    # @test typeof(cons.inner.f.f.second) <: ExaModels.Node2{typeof(*), ExaModels.Var{T1}, T2} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
+    # @test typeof(cons.inner.inner) <: ExaModels.Constraint
+    # @test typeof(cons.inner.inner.f.f) <: ExaModels.Null{Nothing}
+    # @test typeof(cons.inner.inner.inner) <: ExaModels.ConstraintNull
```

Comment:
```suggestion
    jm = JuMP.Model()

    JuMP.@variable(jm, x[1:N])
    JuMP.@variable(jm, p in JuMP.Parameter(1.0))
    JuMP.@constraint(jm, sum(x) == p)
    JuMP.@objective(jm, Min, sum(x))

    em = ExaModel(jm)
    @test only(em.meta.lcon) == only(em.meta.ucon) == 0.0
    @test only(em.θ) == 1.0
    wcons = em.cons
    @test typeof(wcons) <: ExaModels.ConstraintAug
    @test typeof(wcons.f.f) <: ExaModels.Null
    cons = wcons.inner
    @test typeof(cons) <: ExaModels.ConstraintAug
    @test typeof(cons.f.f) <: Pair
    @test typeof(cons.f.f.second) <: ExaModels.Node2{typeof(*), ExaModels.ParameterNode{T1}, T2} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
    @test typeof(cons.inner) <: ExaModels.ConstraintAug
    @test typeof(cons.inner.f.f) <: Pair
    @test typeof(cons.inner.f.f.second) <: ExaModels.Node2{typeof(*), ExaModels.Var{T1}, T2} where {T1<:ExaModels.ParIndexed,T2<:ExaModels.ParIndexed}
    @test typeof(cons.inner.inner) <: ExaModels.Constraint
    @test typeof(cons.inner.inner.f.f) <: ExaModels.Null{Nothing}
    @test typeof(cons.inner.inner.inner) <: ExaModels.ConstraintNull
```
when #158 is merged this should pass